Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -50,7 +50,9 @@ void doChargeDepositionShapeN(const GetParticlePosition& GetPosition,
 #endif
 
     const amrex::Real xmin = xyzmin[0];
+#if (defined WARPX_DIM_3D)
     const amrex::Real ymin = xyzmin[1];
+#endif
```

Comment:
Fixes the warning:
`./Source/Particles/Deposition/ChargeDeposition.H:53:23`: warning: unused variable ‘ymin’ [-Wunused-variable]
     const amrex::Real ymin = xyzmin[1];